graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -132,6 +154,14 @@ def _rename_query_type_fields(ast, renamings, query_type):
     return renamed_ast
 
 
+def _check_for_cascading_type_suppression(
+    ast: DocumentNode, renamings: Dict[str, Optional[str]], query_type: str
+) -> DocumentNode:
```

Comment:
Please make sure all your functions have docstrings. Our rule for docstrings is that if the docstring does not fit on one line, it must have `Args/Returns` sections, and ideally a `Raises` section as well if it raises any notable errors.